python/tvm/relay/expr.py

```diff
@@ -260,6 +260,107 @@ def __init__(self, tuple_value, index):
         self.__init_handle_by_constructor__(
             _make.TupleGetItem, tuple_value, index)
 
+class AbstractExprVisitor(object):
```

Comment:
We should call it AbstractExprFunctor, or ExprFunctor(since abstract doesn't carry too much info)